### ISSUE TITLE
in the Global-Actions-Manager-window: checkbox: double-click versus one click

### ISSUE THREAD

**Issue by ralf3u** [lxqt/lxqt-globalkeys#190]
in the Global-Actions-Manager-window:
at the moment shortcuts can be deactivated in the list of shortcuts by double-click on the checkbox;
the checkbox is exactly the same checkbox like in the properties of the shortcut;
but in the properties only one click on the checkbox is enough to deactivate the shortcut;
that is a contradiction: 2 different behavior for a checkbox;
<b>checkbox</b> should have <b>exactly the same behavior</b>

##### Expected Behavior
in the Global-Actions-Manager-window:
one click to activate/deactivate the checkbox (like the checkbox in the properties of the shortcut, what can be reached for example by double-click on the description of one shortcut)

##### Current Behavior
in the Global-Actions-Manager-window: 
double-click is necessary to activate/deactivate the checkbox

##### Possible Solution
I think it is necessary to add a column on the right side of the checkbox;
in this column there should be only the id-number

the reason:
- checkbox: 1x click for activate/deactivate
- rest of the row, like the id-number: double-click for showing the properties

in other words:
instead of 1 cell with checkbox and id-number, there should be
2 cells, so one cell <b>only</b> for the ckeckbox, and one cell <b>only</b> for the id-number,
because of 2 different behavior

##### Steps to Reproduce (for bugs)
1. open Global-Actions-Manager-window
2. click on the checkbox 1x

##### Context
checkbox should have exactly the same behavior

##### System Information
* Distribution & Version: Lubuntu 20.10
* Qt Version: 0.15.0

**Comment by ralf3u** [comment]
by the way, I think the button "Swap" on the right side of the list of shortcuts does not work (or maybe I missed something);
and the button "Remove" should be deactivated instead of activated when a marked shortcut can't be removed from the list

**Comment by stefonarch** [comment]
Two things should be fixed IMO:
* double click in checkbox (is not intuitive)
* Impossibility to remove some shortcuts nontheless the button "remove" is activated.
  * Alt+Number
  * Cltr+F*
  * xf86Audio*

**Comment by ralf3u** [comment]
@stefonarch "For the latter better an separate issue."

I opened a new issue: https://github.com/lxqt/lxqt-globalkeys/issues/191

**Comment by ralf3u** [comment]
there is also an other issues in Lubuntu 21.04 in LXQt 0.16.0:
if you put Global Actions Manager on a 22'-Full-HD-display in full-screen, there is a horizontal scrollbar although a horizontal scrollbar is not necessary (yes, I'm talking about the horizontal scrollbar, not the vertical scrollbar)
this issue does not appear in the English version but in the German version
